src/player.cpp

```diff
@@ -3963,7 +3964,14 @@ void player::use( item_location loc )
         }
     } else if( used.type->has_use() ) {
         invoke_item( &used, loc.position() );
-
+    } else if( used.has_flag( flag_SPLINT ) ) {
+        ret_val<bool> need_splint = can_wear( used );
+        if( need_splint.success() ) {
+            wear_item( used );
+            loc.remove_item();
+        } else {
+            add_msg( m_info, need_splint.c_str() );
```

Comment:
```suggestion
            add_msg( m_info, need_splint.str() );
```
You don't need a C-String here, it will only be converted to a `std::string` anyway.